linode/manager
# M3-5532: Fixed LKE Stackscripts showing for users

## Description

### The Bug
<img width="900" alt="Screen Shot 2021-10-14 at 12 58 26 AM" src="https://user-images.githubusercontent.com/6440455/137254751-d6f94ec8-8dbf-432e-b138-c41fb55ac76a.png">

- Users were seeing Linode-created Kubernetes Stackscripts that should be hidden (shown in screenshot above)
- This was caused by us now storing Linode's special Kube Debian images in our Redux store
  - This change was made to enhance search functionality so it should be kept
 
### The Fix
- This fix is to not pass any of Linode's kube images to the Stackscript pages which will revert Stackscripts pages back to their old behavior

## How to test

- Have an LKE Cluster spun up on your account
- Go to `/stackscripts/account` and make sure you **don't** see any auto generated LKE Stackscripts
  - Also make sure all other stackscripts show up as expected
